### DISCUSSION TITLE
Intergrate LN content into software-overview.md

### DISCUSSION THREAD

**Discussion by Bosch-0** [BitcoinDesign/Guide#444] [Ideas]
This page currently has no Lightning Network (LN) related content and is one that should not be too difficult to integrate LN into. Below is some proposed suggestions, I am opening this up for discussion before any serious work happens.

### Command-line interfaces (CLI) 

This section can stay the same.

### Block explorers

We should rename this to 'Bitcoin Explorers' and cover both block explorers and Lightning explorers in the description. Hopefully going forward these two services become unified. 

We should update the image to add some LN relevant content here also. Should change the units to sats in the image also.

### Wallets

This section is quite general, so I don't think too many changes are needed here. We don't want to distinguish 'Bitcoin wallet' and 'Lightning wallet' so ensuring LN content added here in a way that does not make that distinction is important. 

I would suggest updating the image to show sats instead of bitcoin.

### Payment processors

No changes needed - though I would update the image used to show sats instead of bitcoin.

### Exchanges

This can stay the same, like above though we should move from bitcoin units to sats in the image. 

### Lightning service providers (LSPs) **(New section)**

LSPs are becoming a pretty common thing with wallets like Breez, Phoenix and Muun pushing the industry in this direction. We should add a section that covers these and what they offer users. 

### Nodes

We should expand this section to mention Lightning nodes alongside the current content. 

### Mining 

Not relevant to LN so no suggestions here.

**Comment by GBKS** [comment]
Thanks for reviewing this page.

For explorers, a big difference is that "Bitcoin explorers" focus on browsing blocks and transactions, while "Lightning explorers" focus on browsing nodes (because transactions are not visible and nodes are unique due to their channels).

I don't see how LSPs fit here, as those are not a category of software but instead entities that provide certain services. Can you maybe elaborate?

**Reply by sbddesign** [reply]
> For explorers, a big difference is that "Bitcoin explorers" focus on browsing blocks and transactions, while "Lightning explorers" focus on browsing nodes (because transactions are not visible and nodes are unique due to their channels).

I agree with @Bosch-0 that LN Explorers should be included in this page. But @GBKS has a good point that they serve fundamentally different purposes.

> Hopefully going forward these two services become unified.

Perhaps. The user needs may be different for a block explorer versus a LN explorer. I'm not opposed to this idea of the 2 merging, but I also am not ready to make this assumption yet.

Maybe better to categorize these two separately? But I think we should definitely include LN explorers.

I agree with Christoph about LSPs, I see them as more of a service provider. I am not sure that LSPs are going to have their own unique UIs...I think of it more like somebody using their wallet's UI to connect to the LSP.

**Reply by Bosch-0** [reply]
> For explorers, a big difference is that "Bitcoin explorers" focus on browsing blocks and transactions, while "Lightning explorers" focus on browsing nodes (because transactions are not visible and nodes are unique due to their channels).

Having two sections for these was my initial idea, but unifying them felt more in line with our attempts to not distinguish the two networks. They are very different though this distinction can be made within the body text, they are both essentially exploring aspects of Bitcoin. Either approach though works fine with me, as you two both want to have a separate section lets go with that option as to not go back on forth on something minor. 

> I don't see how LSPs fit here, as those are not a category of software but instead entities that provide certain services. Can you maybe elaborate?

I was unsure whether to put this here also. I included it as we have exchanges listed here which are mostly also a service (except applications like Bisq, but that's about it). To me, LSPs is a general term that includes any group or individual offering any kind of Lightning service, including services such as lightningnetwork.plug, LNBig etc, which do have UI elements, not just backend services applications like Breez, Phoenix, and Muun offer. 

LSPs will have a crucial role to play in growing the Lightning economy and keeping things decentralized. For example, I could see a decentralized marketplace opening up where individuals could setup their own Pheonix like 'LSP' that users could connect to, say instead of only being able to connect to ACINQ if using Phoenix. 

My thoughts around LSPs / wider Lightning economy this started with this article: https://medium.com/breez-technology/envisioning-lsps-in-the-lightning-economy-832b45871992.

**Comment by GBKS** [comment]
Sorry if it wasn't clear, I am totally on board with one paragraph about explorers, just wanted to point out a difference we might want to explain.

Regarding LSPs, true that exchanges are not exactly a type of software. Reading about [liquidity ads](https://medium.com/blockstream/c-lightning-v0-10-1-eltoo-ethereum-layer-too-2e968a03ca83) today makes me wonder how big LSPs will actually be. Either way, at this moment, it looks like they will play a big role, so I can see us adding them.

**Reply by danielnordh** [reply]
Alternatively, LSP's are mentioned when talking about Lightning nodes in the 'Nodes' section (as they are providing managed nodes).

**Comment by danielnordh** [comment]
Created a doc with the existing content and will start to comment and suggest changes:
https://docs.google.com/document/d/1p534PoTgBfqJvQGX4K2gpOwt8iDoINGitB9Td1Io-3c/edit?usp=sharing

**Reply by pavlenex** [reply]
@danielnordh Awesome, should I assign you to the #465 issue then? 🚀 🚀

**Comment by danielnordh** [comment]
Yes, happy to take this one. I'll assign.
